mtags/src/main/scala-3/scala/meta/internal/pc/ScalaPresentationCompiler.scala

```diff
@@ -177,9 +179,65 @@ case class ScalaPresentationCompiler(
   def implementAbstractMembers(
       params: OffsetParams
   ): CompletableFuture[ju.List[l.TextEdit]] =
-    CompletableFuture.completedFuture(
-      List.empty[l.TextEdit].asJava
-    )
+    val empty: ju.List[l.TextEdit] = new ju.ArrayList[l.TextEdit]()
+    compilerAccess.withInterruptableCompiler(empty, params.token) { pc =>
+      val uri = params.uri
+      val driver = pc.compiler()
+
+      val ctx = driver.currentCtx
+      driver.run(
+        uri,
+        SourceFile.virtual(uri.toASCIIString, params.text)
+      )
+      val unit = driver.currentCtx.run.units.head
+      val tree = unit.tpdTree
+      val pos = driver.sourcePosition(params)
+
+      val newctx = driver.currentCtx.fresh.setCompilationUnit(unit)
+      val path =
+        Interactive.pathTo(newctx.compilationUnit.tpdTree, pos.span)(using
+          newctx
+        )
+
+      val indexedContext = IndexedContext(
+        MetalsInteractive.contextOfPath(path)(using newctx)
+      )
+      import indexedContext.ctx
+
+      val autoImportsGen = AutoImports.generator(
+        pos,
+        params.text,
+        unit.tpdTree,
+        indexedContext,
+        config
+      )
+
+      object ObjectCreation:
```

Comment:
Not sure about the purpose here. Maybe
```suggestion
      object FindTypeDef:
```

Reply:
https://github.com/scalameta/metals/pull/3960/commits/7dceb05f286bf11b94c4804a9a03dd491208d15f :+1: